pytorch_lightning/plugins/training_type/sharded.py

```diff
@@ -85,6 +86,11 @@ def _optim_state_dict(self, optimizer):
 
     @property
     def lightning_module(self) -> LightningModule:
+        if not _FAIRSCALE_AVAILABLE:  # pragma: no cover
+            raise MisconfigurationException(
+                "`DDPShardedPlugin` requires `fairscale` to be installed."
+                " Install it by running `pip install fairscale`."
```

Comment:
could be single line so less untested lines :]

Reply:
@Borda I already added   `# pragma: no cover`